code/datums/wires/airlock.dm

```diff
@@ -55,9 +55,12 @@ var/const/AIRLOCK_WIRE_LIGHT = 512
 /datum/wires/airlock/CanUse(mob/living/L)
 	var/obj/machinery/door/airlock/A = holder
 	if(iscarbon(L))
-		if(A.isElectrified())
-			if(A.shock(L, 100))
-				return 0
+		if(A.x - L.x <= 1 && A.x - L.x >= -1 && A.y - L.y <= 1 && A.y - L.y >= -1)
```

Comment:
Use Adjacent and early return, i.e.

`if(!A.Adjacent(L)
    return 0
if(A.isElectrified())
    if(A.shock(L, 100))
        return 0
`

Reply:
Title was fixed. It was vague.
I didn't use Adjacent due to a small misunderstanding regarding how it works, but it seems to do its job just as fine, if not better thanks to readability.
Everything was tested locally and worked exactly as I expected it to.